docs/src/pages/customization/breakpoints/breakpoints-de.md

```diff
@@ -12,9 +12,9 @@ Jeder Haltepunkt (ein Schlüssel) stimmt mit einer *festen* Bildschirmbreite (ei
 
 - ** xs, ** extraklein: 0px
 - ** sm, ** klein: 600px
-- ** md, ** mittel: 900px
-- ** lg, ** groß: 1200px
-- ** xl ** extra groß: 1536px
```

Comment:
Is this not correct?

Reply:
These changes are lost as soon as we sync with Crowdin. It makes the review more daunting.

Reply:
I see, will keep that in mind.

Reply:
See a sync example: #26768